lib/features/push_notification/presentation/controller/push_base_controller.dart

```diff
@@ -60,7 +60,7 @@ abstract class PushBaseController {
       .toList();
 
     final listEmailActions = listTypeName
-      .where((typeName) => typeName == TypeName.emailType || typeName == TypeName.emailDelivery)
+      .where((typeName) => typeName == TypeName.emailDelivery)
```

Comment:
Looks like you made a mistake here. `_toPushNotificationAction` is a function that converts to many actions, not just `PushNotification`. Please check it again carefully.

<img width="982" alt="Screenshot 2025-01-02 at 14 33 43" src="https://github.com/user-attachments/assets/3b0ea2bf-7fa1-4912-991a-717808fa7a01" />